### ISSUE TITLE
When I try to run the code, I get: Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by rahamin1** [shentaoy/react-navigation-material-drawer-example#1]
I cloned the project , then did `npm install`, then `react-native run-android` and got the following error (I tried also `yarn install`, same error) :

```
bundling failed: Error: Unable to resolve module `AccessibilityInfo`
from `C:\esites-native\react-navigation\react-navigation-material-drawer-example\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: 
Module `AccessibilityInfo` does not exist in the Haste module map
```

Any idea why this happens?
